build.gradle.kts

```diff
@@ -13,8 +13,8 @@ plugins {
   alias(libs.plugins.versionsFilter)
 }
 
-fun getFlagsForAddOpens(vararg packages: String, module: String): List<String> {
-  return packages.toList().map { "--add-opens=$module/$it=ALL-UNNAMED" }
```

Comment:
`opens` is a superset of `exports`, wasn't needed here (https://stackoverflow.com/a/44059335)